app/index.html

```diff
@@ -85,6 +85,11 @@
         </div>
         <div class="text">Give Feedback</div>
       </div>
+
+      <a class="menu-option" href="/">
```

Comment:
`href="/"` will always reload the page. We can do `onclick="pauseAll()"` instead to pause all.